expandybird/Makefile

```diff
@@ -1,17 +1,22 @@
-# Makefile for the Docker image gcr.io/$(PROJECT)/expandybird
+# Makefile for the Docker image $(DOCKER_REGISTRY)/$(PROJECT)/expandybird
 # MAINTAINER: Jack Greenfield <jackgr@google.com>
 # If you update this image please check the tag value before pushing.
 
 .PHONY : all build test push container clean
 
-PREFIX := gcr.io/$(PROJECT)
+DOCKER_REGISTRY := gcr.io
```

Comment:
Yes, this could be completed by overriding the Makefile variable when calling `make push`

Example of push to Docker Hub:
```
make push DOCKER_REGISTRY=index.docker.io PROJECT=(your account)
```